pylint/checkers/variables.py

```diff
@@ -2329,6 +2339,45 @@ def _check_classdef_metaclasses(self, klass, parent_node):
 
         return consumed
 
+    def _node_should_be_skipped(
+        self, consumer: NamesConsumer, node: nodes.Name, is_start_index: bool
+    ) -> bool:
```

Comment:
Why did you move that method to the bottom? Wouldn't it make sense between `visit_name` and `_check_consumer`?

Reply:
Not sure, moved it to above `_check_consumer`